Rakefile

```diff
@@ -11,29 +11,9 @@ rescue Bundler::BundlerError => e
   exit e.status_code
 end
 
-require 'rake'
-require 'jeweler'
```

Comment:
Since you removed this, shouldn't you also remove jeweler as gem requirement?

Reply:
@weshatheleopard yup, good call. I've removed it.